mobile/src/bundles/common/components/header/styles.ts

```diff
@@ -0,0 +1,29 @@
+import { StyleSheet } from 'react-native';
+
+import { Color } from '~/bundles/common/enums/enums';
+
+const styles = StyleSheet.create({
+    header: {
+        backgroundColor: Color.TEXT,
+        height: 100,
```

Comment:
```suggestion
```
globalStyles

Reply:
I need to create this style? Because we do not have it there

Reply:
We have height 100 style 
`style={[styles.header, globalStyles.h100]}`

Reply:
We have only height100 which is 100%
![image](https://github.com/BinaryStudioAcademy/bsa-2023-bsa-talents/assets/104503014/6bcac405-ed7a-4e2b-b0a3-60b1a74a980d)

Reply:
Oh, sorry, I thought we have % there, then leave it as it